src/test/java/io/vertx/cassandra/StreamingTest.java

```diff
@@ -0,0 +1,44 @@
+package io.vertx.cassandra;
+
+import io.vertx.core.Future;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.regex.Pattern;
+
+@RunWith(VertxUnitRunner.class)
+public class StreamingTest extends CassandraServiceBase {
```

Comment:
In the test you must make sure that:

- no items are emitted when you invoke `pause`, until you call `resume`, see https://github.com/vert-x3/vertx-infinispan/blob/master/src/test/java/io/vertx/ext/cluster/infinispan/test/InfinispanClusteredAsyncMapTest.java#L116
- no items are emitted after the stream is closed (by setting a `null` handler), see https://github.com/vert-x3/vertx-infinispan/blob/master/src/test/java/io/vertx/ext/cluster/infinispan/test/InfinispanClusteredAsyncMapTest.java#L151